doc/templates/index.html

```diff
@@ -156,7 +156,7 @@ <h4 class="sk-landing-call-header">News</h4>
         <li><strong>On-going development:</strong>
         <a href="https://scikit-learn.org/dev/whats_new.html"><strong>What's new</strong> (Changelog)</a>
         </li>
-        <li><strong>February 2020.</strong> scikit-learn 0.22.2 is available for download (<a href="whats_new/v0.22.html#version-0-22-2">Changelog</a>).
+        <li><strong>March 2020.</strong> scikit-learn 0.22.2 is available for download (<a href="whats_new/v0.22.html#version-0-22-2">Changelog</a>).
```

Comment:
Should we say "scikit-learn 0.22.2.post1" here ?

Reply:
I'm happy either way.

Reply:
Let's not put post here. It may lead to confusion.

Reply:
You could just say Fixed source distribution available from March

Reply:
Sorry don't get what you're proposing. Can you make it as a suggestion ?

Reply:
Sorry, I missed this ongoing discussion and already merged the PR. I believe this is fine as it is right?

Reply:
We can always update the website after the 0.22.2.post1 release by pushing to 0.22.X after the tag anyway.